mlcustomize/inject_virtio_win.ml

```diff
@@ -474,8 +474,8 @@ and virtio_iso_path_matches_guest_os t path =
 
     (* Using the full path, work out what version of Windows
      * this driver is for.  Paths can be things like:
-     * "NetKVM/2k12R2/amd64/netkvm.sys" or
-     * "./drivers/amd64/Win2012R2/netkvm.sys".
+     * "$ISO/NetKVM/2k12R2/amd64/netkvm.sys" or
+     * "/usr/share/virtio-win/drivers/by-os/amd64/2k12R2/netkvm.sys".
```

Comment:
Is it true that `$ISO` or `/usr/share/virtio-win` can literally appear in the path here?

The code is rather hard to follow to say the least, but looking at `copy_from_virtio_win`, if passed a directory then we `cd` into the directory and run `find` to get the list of paths, so they'll be relative, and begin with `./` (as the original comment said).  If passed a file (ie. the ISO) then we open the ISO with libguestfs and use `g#find` to list the paths, which returns the raw path relative to the root of the ISO mountpoint.

So I believe the original comment was correct here, at least the prefix part.

Reply:
Ok will fix